web/frontend/src/pages/election/components/ElectionForm.tsx

```diff
@@ -29,19 +29,18 @@ const ElectionForm: FC<ElectionFormProps> = ({ setShowModal, setTextModal }) =>
   // conf is the configuration object containing MainTitle and Scaffold which
   // contains an array of subject.
   const { t } = useTranslation();
-  const UserID = sessionStorage.getItem('id');
   const emptyConf: Configuration = emptyConfiguration();
   const [conf, setConf] = useState<Configuration>(emptyConf);
   const { MainTitle, Scaffold } = conf;
 
   async function createHandler() {
     const data = {
       Configuration: marshalConfig(conf),
-      UserID,
     };
     const req = {
       method: 'POST',
       body: JSON.stringify(data),
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
By adding that line, we cannot longer create an election, to make it work again in the mock handler we should change the parsing of the body from `const body: NewElectionBody = JSON.parse(req.body.toString())` to 
`const body = req.body as   NewElectionBody`